src/utils/getRandomNumber.js

```diff
@@ -0,0 +1,27 @@
+import NumberValidator from './validators/NumberValidator.js';
+
+/**
+ * 주어진 범위 내에서 랜덤 숫자를 반환하는 함수
+ *
+ * @param {number} num1
+ * @param {number} num2
+ *
+ * @example
+ * // 0부터 10까지의 랜덤 숫자 반환
+ * getRandomNumber(0, 10);
+ *
+ * @example
+ * // 5부터 15까지의 랜덤 숫자 반환
+ * getRandomNumber(5, 15);
+ */
+function getRandomNumber(num1, num2) {
+  let random;
+
+  NumberValidator.from(num1).lessThanOrEqual(num2);
+  NumberValidator.from(num2);
+
+  random = Math.floor(Math.random() * (num2 - num1 + 1)) + num1;
+  return parseInt(random);
+}
```

Comment:
```suggestion
function getRandomNumber(num1, num2) {
  NumberValidator.from(num1).lessThanOrEqual(num2);
  NumberValidator.from(num2);

  return Math.floor(Math.random() * (num2 - num1 + 1)) + num1;
}
```

어차피 floor를 통해서 소숫점이 날라가서 이렇게 해도 되지 않을까 싶어요!

아마 num1과 num2가 소수일 수도 있는 상황을 가정한건가 싶기도 하고...